tests/README.md

```diff
@@ -91,14 +103,17 @@ For example look at the [block long haul test](/tests/block/k8s/longhaul/basicBl
  To run a longhaul test you can run any integration test with `-count` and `--load_parallel_runs` options
  e.g.
  ```
- go test -run TestK8sBlockLongHaul github.com/rook/rook/e2e/tests/block/k8s/longhaul --load_parallel_runs=20 -count=1000
+ go test -run TestK8sBlockLongHaul github.com/rook/rook/tests/longhaul --load_parallel_runs=20 -count=1000
  ```
+ The Longhaul test just unlike other test is going to install if its is not already installed, but it is not going to clean up test data or uninstall rook after the run. 
```

Comment:
cleaned up, the typos, I had a spell checker plugin installed, but the IDE editor color scheme was making it difficult to catch typos. I have changed the color scheme so that typos are more visible.